### PR TITLE
WordBazaar drop abbreviation

### LINKED ISSUE #1032: Create token WB adjustments
- remove `abbreviation`
- limit name to `14` chars and pass a copy of it in the ticker to be compliant with AEX9

### PR DESCRIPTION
**Pull request by AtanasKrondev**
Closes #1032 

- Removed abbreviation
- Name limited to 14 symbols and a copy passed as a ticker when creating
- Some design adjustments to fit

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
In a web browser: 
If the browser is not expanded to full view, then this bug appears:
err1

After creating a vote:
token name lenght
revoke

In a mobile browser:
when scrolling up, some elements become visible (go over borders):


1111 jpeg

The word "token" is truncated in the description, you need to scroll down to see it:


Some pop-ups are in the wrong position
https://user-images.githubusercontent.com/69896204/110782045-4f660980-826f-11eb-92b4-7bd87aeba511.mp4

**Comment by Liubov-crypto** [comment]
Rechecked, found this bug in a mobile browser: 

https://user-images.githubusercontent.com/69896204/110831935-6cb4cb00-82a3-11eb-9001-0965bf8329ee.mp4

I can't buy FT with an iframe wallet:

https://user-images.githubusercontent.com/69896204/110832039-8b1ac680-82a3-11eb-9850-57cff110f57f.mov

 When I put .chain name instead of wallet address by creating a vote  I got this error:
chain name
console

**Comment by Liubov-crypto** [comment]
Rechecked, the buy/sell modal when scrolling, now LGTM.

**Comment by mradkov** [comment]
Screenshot 2021-03-12 at 11 52 18

14x `W`

File: src/components/CreateToken.vue [hunk 1]
```diff
@@ -195,14 +164,13 @@ export default {
...
     invalidInputs() {
       return this.loadingState
-      || !this.name.length || !this.description.length || !this.abbreviation.length
+      || !this.name.length || !this.description.length
       || Object.values(this.error).includes((e) => e !== null);
     },
     error() {
       return {
-        name: this.name.length > 333 ? this.$t('components.CreateToken.Name.Error') : null,
+        name: this.name.length > 14 ? this.$t('components.CreateToken.Name.Error') : null,
```
Comment (davidyuk) [inline src/components/CreateToken.vue:172]: Would be nice to extract `14`, I'm sure it is used in several places

File: src/components/PriceHistory.vue [hunk 2]
```diff
@@ -49,6 +49,9 @@ export default {
         min: this.daysAgo === 0 ? 0 : start,
       };
 
+      const maxYTick = this.formatedData('Buy')
+        .reduce((prev, current) => ((prev.y > current.y) ? prev : current)).y * 1.1;
```
Comment (davidyuk) [inline src/components/PriceHistory.vue:53]: Proposing a shorter way:
```suggestion
      const maxYTick = Math.max(...this.formatedData('Buy').map(({ y }) => y)) * 1.1;
```

File: src/locales/en.json [hunk 3]
```diff
@@ -648,22 +647,15 @@
       "Name": {
         "Title": "Token",
         "Placeholder": "Enter any combination of characters *",
-        "Counter": " / 333",
-        "Error": "Maximum number of characters allowed is 333"
+        "Counter": " / 14",
```
Comment (davidyuk) [inline src/locales/en.json:650]: is `/` going to be translated to other languages?

File: src/locales/en.json [hunk 4]
```diff
@@ -648,22 +647,15 @@
...
+        "Error": "Maximum number of characters allowed is 14"
```
Comment (davidyuk) [inline src/locales/en.json:651]:
```suggestion
        "Error": "Maximum number of characters allowed is {maxNameLength}"
```

**Review by thepiwo — APPROVED** [review]

**Pull request merged**